### DISCUSSION TITLE
v4.3.0 release

### DISCUSSION THREAD

**Discussion by andyzhangx** [kubernetes-csi/csi-driver-nfs#455] [Announcements]
### Features 🌈
 - enable securityContext.seccompProfile (#437)
 - support volume cloning (#426)
 - support snapshot (#430)
 - provide option to retain subdir on delete (#432)

### Bug Fixes 🐞
 - fix: use mkdir -p for creating subdir in mounted base-dir (#412)
 - fix: CVE-2022-41723 (#415)
 - fix: CVE-2022-3294 (#435)
 - fix: set ClusterFirstWithHostNet as default dnsPolicy (#440)

## What's Changed
* doc: cut v4.2.0 release by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/401
* test: update release tools by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/417
* Update debian-base image to v1.4.3 by @saschagrunert in https://github.com/kubernetes-csi/csi-driver-nfs/pull/419
* test: fix golint error by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/425
* feat: volume clone from source volume by @wozniakjan in https://github.com/kubernetes-csi/csi-driver-nfs/pull/426
* feat: add readOnlyRootFilesystem if possible by @farodin91 in https://github.com/kubernetes-csi/csi-driver-nfs/pull/422
* test: copy volume from volume unit tests by @wozniakjan in https://github.com/kubernetes-csi/csi-driver-nfs/pull/433
* fix: CVE-2022-3294 by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/435
* feat: provide option to retain subdir on delete by @woehrl01 in https://github.com/kubernetes-csi/csi-driver-nfs/pull/432
* feat: implement snapshots as tarballs by @wozniakjan in https://github.com/kubernetes-csi/csi-driver-nfs/pull/430
* feat: enable securityContext.seccompProfile, set system-cluster-critical by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/437
* test(e2e): change the storageclass configuration method in the test file by @umagnus in https://github.com/kubernetes-csi/csi-driver-nfs/pull/438
* fix: set ClusterFirstWithHostNet as default dnsPolicy by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/440
* chore: upgrade csi sidecar container images by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/441
* chore: add dependabot by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/442
* chore(deps): bump github/codeql-action from 1 to 2 by @dependabot in https://github.com/kubernetes-csi/csi-driver-nfs/pull/443
* chore(deps): bump sigs.k8s.io/yaml from 1.2.0 to 1.3.0 by @dependabot in https://github.com/kubernetes-csi/csi-driver-nfs/pull/444
* chore(deps): bump actions/setup-go from 2 to 4 by @dependabot in https://github.com/kubernetes-csi/csi-driver-nfs/pull/445
* chore(deps): bump golang.org/x/net from 0.7.0 to 0.9.0 by @dependabot in https://github.com/kubernetes-csi/csi-driver-nfs/pull/446
* chore(deps): bump google.golang.org/protobuf from 1.27.1 to 1.30.0 by @dependabot in https://github.com/kubernetes-csi/csi-driver-nfs/pull/447
* chore(deps): bump github.com/onsi/gomega from 1.10.1 to 1.27.6 by @dependabot in https://github.com/kubernetes-csi/csi-driver-nfs/pull/448
* Use dl.k8s.io instead of hardcoded GCS URIs by @stmcginnis in https://github.com/kubernetes-csi/csi-driver-nfs/pull/450
* chore: upgrade snapshot version and refine snapshot deployment by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/451
* chore: upgrade csi-snapshot to v6.2.2 by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/453

## New Contributors
* @saschagrunert made their first contribution in https://github.com/kubernetes-csi/csi-driver-nfs/pull/419
* @farodin91 made their first contribution in https://github.com/kubernetes-csi/csi-driver-nfs/pull/422
* @woehrl01 made their first contribution in https://github.com/kubernetes-csi/csi-driver-nfs/pull/432
* @dependabot made their first contribution in https://github.com/kubernetes-csi/csi-driver-nfs/pull/443
* @stmcginnis made their first contribution in https://github.com/kubernetes-csi/csi-driver-nfs/pull/450

**Full Changelog**: https://github.com/kubernetes-csi/csi-driver-nfs/compare/v4.2.0...v4.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kubernetes-csi/csi-driver-nfs/releases/tag/v4.3.0'>v4.3.0 release</a>.</em>

**Comment by mygpse** [comment]
oh, no success update via kubeapps. 4.2.0 -> 4.3.0
![image](https://github.com/kubernetes-csi/csi-driver-nfs/assets/3582197/88fae937-e446-4888-8dd7-aa142e18806f)

**Reply by andyzhangx** [reply]
the v4.3.0 image publish depends on https://github.com/kubernetes/k8s.io/pull/5346 merge

**Reply by ma-gokbas** [reply]
Failed to pull image "registry.k8s.io/sig-storage/nfsplugin:v4.3.0": rpc error: code = NotFound desc = failed to pull and unpack image "registry.k8s.io/sig-storage/nfsplugin:v4.3.0": failed to resolve reference "registry.k8s.io/sig-storage/nfsplugin:v4.3.0": registry.k8s.io/sig-storage/nfsplugin:v4.3.0: not found

**Reply by andyzhangx** [reply]
registry.k8s.io/sig-storage/nfsplugin:v4.3.0 is published now
